test/common/config.js

```diff
@@ -1,19 +1,43 @@
+import { format as formatUrl } from 'url';
+import { OPTIMIZE_BUNDLE_DIR, esTestConfig, kbnTestConfig } from '@kbn/test';
 import {
   KibanaServerProvider,
   EsProvider,
   EsArchiverProvider,
   RetryProvider,
 } from './services';
 
-import { esTestConfig } from '../../src/test_utils/es';
-import { kibanaTestServerUrlParts } from '../kibana_test_server_url_parts';
-
 export default function () {
+  const servers = {
+    kibana: kbnTestConfig.getUrlParts(),
+    elasticsearch: esTestConfig.getUrlParts(),
+  };
+
   return {
-    servers: {
-      kibana: kibanaTestServerUrlParts,
-      elasticsearch: esTestConfig.getUrlParts(),
+    servers,
+
+    esTestCluster: {
+      license: 'oss',
+      from: 'snapshot',
+      serverArgs: [
+      ],
     },
+
+    kibanaServerArgs: [
+      '--env=development',
```

Comment:
__question/issue:__ Shouldn't we define `--oss` here?

Reply:
I think I took care of this issue now. from kibana root, doing `node scripts/functional_tests --config test/api_integration.config` should now be working again.